### ISSUE TITLE
[Bug Report] Sparkline doesn't support label-size for 'trend' type

### ISSUE THREAD

**Issue by kerf007** [vuetifyjs/vuetify#6672]
### Versions and Environment
**Vuetify:** 1.5.4
**Vue:** 2.6.8
**Browsers:** Chrome 72.0.3626.119
**OS:** Mac OS 10.14.3

### Steps to reproduce
Set `label-size` attr for `trend` type to any value

### Expected Behavior
Font size for labels must be changed

### Actual Behavior
Font size for labels doesn't change

### Reproduction Link
<a href="https://codepen.io/kerf/pen/VRmLzO" target="_blank">https://codepen.io/kerf/pen/VRmLzO</a>

**Comment by rkacenski** [comment]
Still a bug on 2.0.0-beta.0

**Comment by guykatz** [comment]
code explicitly uses label-size only for bar type label generation for some reason. 
trend labels inherit their size from their g element which is a hard coded 8px in the code.
can be bypassed via css.

**Comment by vw-bmacher** [comment]
A little workaround: 
`<v-sparline ref="sparkline" ...></sparkline>`

```javascript
mounted() {
  const textElements = this.$refs.sparkline.$el.querySelectorAll('text');
  for (const textElement of textElements) {
    textElement.style.fontSize = '8px';
  }
}
```

**Comment by nekosaur** [comment]
Fixed in #7642
